src/minesweeper/use-cases/startGame/startGame.ts

```diff
@@ -24,36 +23,22 @@ type IBoardResponse = Omit<BoardModel, 'bombs' | 'neighBorsBombsCounter'>;
 type IStartGameUseCase = (props: IStartGameProps) => Promise<IBoardResponse>;
 
 /**
- * start game base on initial props
+ * Start game base on initial props
  */
-
 export const startGameUseCase: IStartGameUseCase = async ({
   bombs: bombsInput,
   rows,
   columns,
-  data,
+  dataRepository,
 }) => {
-  if (bombsInput < 1) {
-    throw Error(ErrorStartGame.BOMBS_GREATER_THAN_ZERO);
-  }
-
-  if (rows < 1) {
-    throw Error(ErrorStartGame.ROWS_GREATER_THAN_ZERO);
-  }
-
-  if (columns < 1) {
-    throw Error(ErrorStartGame.COLUMNS_GREATER_THAN_ZERO);
-  }
-
-  if (rows < 2 && columns < 2) {
-    throw new Error(ErrorStartGame.ROWS_AND_COLUMNS_GREATER_THAN_ONE);
-  }
+  if (bombsInput < 1) throw Error(ErrorStartGame.BOMBS_GREATER_THAN_ZERO);
+  if (rows < 1) throw Error(ErrorStartGame.ROWS_GREATER_THAN_ZERO);
+  if (columns < 1) throw Error(ErrorStartGame.COLUMNS_GREATER_THAN_ZERO);
+  if (rows < 2 && columns < 2) throw new Error(ErrorStartGame.ROWS_AND_COLUMNS_GREATER_THAN_ONE);
```

Comment:
Not sure if we agree in this style of the `if`

Reply:
If is only for throw error it's ok